src/sqs/deployment.ts

```diff
@@ -1,9 +1,33 @@
 import { WebhookPayloadDeploymentStatus } from "@octokit/webhooks";
+import { Context, MessageHandler } from "./index";
+import app from "../worker/app";
+import { processDeployment } from "../github/deployment";
 
 export type DeploymentMessagePayload = {
 	jiraHost: string,
+	installationId: number,
+	webhookReceived: number,
+	webhookId: string,
 
 	// The original webhook payload from GitHub. We don't need to worry about the SQS size limit because metrics show
 	// that payload size for deployment_status webhooks maxes out at 13KB.
-	webhookPayload: WebhookPayloadDeploymentStatus
+	webhookPayload: WebhookPayloadDeploymentStatus,
```

Comment:
In some cases (like pullrequests with a big description), we have to be conscious of message size, because SQS messages have a limit of 256KB. But `deployment_status` events are very small.